components/grid/grid.css

```diff
@@ -0,0 +1,46 @@
+@import url('../../components/framer.css');
+
+.grid-equal{
+   display: grid;
+   grid-template-columns: repeat(3,1fr);
+   gap:10px;
+}
+.grid-50-50{
+    
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap:10px;
+}
+
+.grid-70-30{
+    
+    display: grid;
+    grid-template-columns: 7fr 3fr;
+    gap:10px;
+}
+
+.grid-30-70{
+    
+    display: grid;
+    grid-template-columns: 3fr 7fr;
+    gap:10px;
+}
+
+.items{
+    height: 450px;
+    background: var(--col-dark);
+}
+
+.img-cover{
+    width: 100%;
+    height: 100%;
```

Comment:
Using these properties only image spread throughout the container (without keeping its original aspect ratio) but distorted if container width is too low and height is too high. Instead of it, you can use object-fit: cover property for setting image as a cover. I am not sure your actual need, if this is just showing grid layout then it's okay.